src/models/StyledComponent.js

```diff
@@ -219,6 +219,8 @@ export default (ComponentStyle: Function) => {
     options: Object,
     rules: RuleSet
   ) {
+    const targetIsSC = isStyledComponent(target)
```

Comment:
nit: Shall we improve the variable name here? e.g. `isTargetStyledComp` or sth?

Reply:
Yeah makes sense